test/test_package.py

```diff
@@ -38,5 +39,7 @@ def test_show_package_not_found(db, client):
 @create_advisory(id=DEFAULT_ADVISORY_ID, group_package_id=DEFAULT_GROUP_ID, advisory_type=issue_types[1], reference='https://security.archlinux.org', publication=Publication.published)
 def test_show_package(db, client):
     resp = client.get(url_for('tracker.show_package', pkgname='foo'), follow_redirects=True)
+    html = AssertionHTMLParser()
+    html.feed(resp.data.decode())
```

Comment:
is there some assertion missing or was the `AssertionHTMLParser` added by accident?

Reply:
it tests that you can feed the page to the parser without raising an exception.
It was previously failing.